### PR TITLE
Feature/977 HTML structure

### PR DESCRIPTION
**Pull request by ManuelMoeri**

### CODE REVIEW

File: frontend/src/app/components/keyresult-detail/keyresult-detail.component.html [hunk 1]
```diff
@@ -98,28 +98,30 @@ <h2 class="sub-title-keyresult-detail fw-bold mb-1">
...
-        <div class="mt-3 mb-4">
-          <h2 class="sub-title-keyresult-detail fw-bold mb-1">Beschrieb</h2>
-          <p class="linebreak" *ngIf="keyResult.description === ''">-</p>
-          <p class="linebreak" *ngIf="keyResult.description !== ''">{{ keyResult.description }}</p>
-        </div>
+        <section>
+          <div class="mt-3 mb-4">
+            <h3 class="sub-title-keyresult-detail fw-bold mb-1">Beschrieb</h3>
+            <p class="linebreak" *ngIf="keyResult.description === ''">-</p>
+            <p class="linebreak" *ngIf="keyResult.description !== ''">{{ keyResult.description }}</p>
```
Comment (kcinay055679) [inline frontend/src/app/components/keyresult-detail/keyresult-detail.component.html:107]: probably merge this tag using the or syntax (keyResult.description || "-")

File: frontend/src/app/components/keyresult-detail/keyresult-detail.component.html [hunk 2]
```diff
@@ -98,28 +98,30 @@ <h2 class="sub-title-keyresult-detail fw-bold mb-1">
...
-              <div class="action-list-checked-item break-word ms-4">
-                <img src="../../../assets/icons/checkmark.svg" alt="checkbox" class="action-checkbox me-1" />
-                <span [ngStyle]="{ 'text-decoration': 'line-through' }"> {{ action.action }}</span>
+          <div *ngIf="keyResult.actionList?.length != 0" class="mt-3 mb-4">
+            <h3 class="sub-title-keyresult-detail fw-bold">Action Plan</h3>
+            <div *ngFor="let action of keyResult.actionList">
+              <div *ngIf="action.isChecked">
+                <div class="action-list-checked-item break-word ms-4">
+                  <img src="../../../assets/icons/checkmark.svg" alt="checkbox" class="action-checkbox me-1" />
+                  <span [ngStyle]="{ 'text-decoration': 'line-through' }"> {{ action.action }}</span>
```
Comment (kcinay055679) [inline frontend/src/app/components/keyresult-detail/keyresult-detail.component.html:116]: why span instead of p?

File: frontend/src/app/components/overview/overview.component.html [hunk 3]
```diff
@@ -1,5 +1,5 @@
 <app-application-banner id="bannerComponent"></app-application-banner>
-<div id="overview" class="overviewContainer px-sm-4" [ngStyle]="{ 'padding-top.px': overviewPadding | async }">
+<div id="overview" class="overview-container px-sm-4" [ngStyle]="{ 'padding-top.px': overviewPadding | async }">
```
Comment (kcinay055679) [inline frontend/src/app/components/overview/overview.component.html:2]: why did you change the class name?
Accoring to Google you should use kebab-case for css classes

File: frontend/src/app/components/team/team.component.html [hunk 4]
```diff
@@ -1,10 +1,10 @@
 <div *ngIf="overviewEntity as OVEntity" class="bg-grey justify-content-between pt-3">
   <section class="team-header">
-    <span class="d-flex align-items-center gap-2">
+    <div class="d-flex align-items-center gap-2">
```
Comment (kcinay055679) [inline frontend/src/app/components/team/team.component.html:3]: why two divs?

File: frontend/src/app/shared/custom/scoring/scoring.component.html [hunk 5]
```diff
@@ -1,16 +1,14 @@
...
-          <span class="text-center text-small">Fail</span>
-        </span>
-        <span class="okr-score-label okr-score-commit">
-          <span class="text-center text-small">Commit</span>
-        </span>
-        <span class="okr-score-label">
-          <span class="text-center text-small">Target</span>
-        </span>
+      <div class="okr-score-label">
+        <h5 class="text-center text-small">Fail</h5>
```
Comment (kcinay055679) [inline frontend/src/app/shared/custom/scoring/scoring.component.html:5]: do we really need the parent div around each element?

File: frontend/src/app/team-management/invite-user-dialog/invite-user-dialog.component.html [hunk 6]
```diff
@@ -1,8 +1,8 @@
 <section mat-dialog-title>
   <div class="d-flex justify-content-between align-items-center position-relative dialog-header mt-1">
-    <span class="title w-100 mb-2">
+    <div class="title w-100 mb-2">
```
Comment (kcinay055679) [inline frontend/src/app/team-management/invite-user-dialog/invite-user-dialog.component.html:3]: Why did you not use an `h` Tag?
But doesnt really mattter since this will be replaced in #1072

**Review by kcinay055679 — CHANGES_REQUESTED** [review]
Look again over the following spots:

- [x] confidence label on overview
- [ ] Checkin-history component 
- [ ] Stretch label on scoring component
- [ ] Review all ussages of section

File: frontend/src/app/components/check-in-history-dialog/check-in-history-dialog.component.html [hunk 7]
```diff
@@ -47,7 +47,7 @@
           <span class="label-width"><i>Massnahmen: </i></span>
```
[the remark this answers is not in the document]
Comment (ManuelMoeri) [reply frontend/src/app/components/check-in-history-dialog/check-in-history-dialog.component.html:47]: good call

File: frontend/src/app/components/keyresult-detail/keyresult-detail.component.html (hunk 2 above)
Comment (ManuelMoeri) [reply frontend/src/app/components/keyresult-detail/keyresult-detail.component.html:116]: does not really make a difference imo

File: frontend/src/app/components/team/team.component.html (hunk 4 above)
Comment (ManuelMoeri) [reply frontend/src/app/components/team/team.component.html:3]: second div has been deleted.

File: frontend/src/app/shared/custom/scoring/scoring.component.html (hunk 5 above)
Comment (ManuelMoeri) [reply frontend/src/app/shared/custom/scoring/scoring.component.html:5]: Having wrappers may unfortunately be mandatory here. I will replace them with a `span` since they have `inline-grid` styling applied.

File: frontend/src/app/components/objective/objective.component.spec.ts [hunk 8]
```diff
@@ -19,6 +19,7 @@ import { MatTooltipModule } from '@angular/material/tooltip';
 import { ScoringComponent } from '../../shared/custom/scoring/scoring.component';
 import { ConfidenceComponent } from '../confidence/confidence.component';
 import { ReactiveFormsModule } from '@angular/forms';
+// @ts-ignore
```
Comment (peggimann) [inline frontend/src/app/components/objective/objective.component.spec.ts:22]: this is dangerous please remove

File: frontend/src/app/shared/dialog/complete-dialog/complete-dialog.component.html [hunk 9]
```diff
@@ -38,12 +38,12 @@
...
           >
             Objective nicht erreicht
-          </div>
+          </button>
         </div>
       </div>
 
       <div class="input-style gap-2 col-12">
-        <label for="comment" class="text-black">Kommentar (optional)</label>
+        <h4 for="comment" class="text-black">Kommentar (optional)</h4>
```
Comment (peggimann) [inline frontend/src/app/shared/dialog/complete-dialog/complete-dialog.component.html:46]: this should be a label

**Review by peggimann — CHANGES_REQUESTED** [review]
Overall nice Job! I mainly see some possible improvements in changing Labels to the label tag. Otherwise i only found some nitpicks :D

**Review by peggimann — APPROVED** [review]

**Review by peggimann — APPROVED** [review]
